cmd/osbuild-worker-executor/handler_build.go

```diff
@@ -39,63 +41,96 @@ func (wf *writeFlusher) Write(p []byte) (n int, err error) {
 	return n, err
 }
 
-func runOsbuild(logger *logrus.Logger, buildDir string, control *controlJSON, output io.Writer) (string, error) {
-	flusher, ok := output.(http.Flusher)
-	if !ok {
-		return "", fmt.Errorf("cannot stream the output")
-	}
-	// stream output over http
```

Comment:
This is probably an extremly stupid question (sorry!) - why do we need this change? What would happen if we just stream the json-seq in exactly the same way as we used to stream the (previous) text output?